packages/@aws-cdk/aws-iotevents/test/detector-model.test.ts

```diff
@@ -109,6 +109,7 @@ test('can set multiple events to State', () => {
         {
           eventName: 'test-eventName1',
           condition: iotevents.Expression.fromString('test-eventCondition'),
+          actions: [{ bind: () => ({ configuration: { setTimer: { timerName: 'test-timer' } } }) }],
```

Comment:
Let's avoid touching existing tests unless strictly necessary. Let's add new tests that deal with Actions.

Same comment everywhere in this file.